content/docs/studio/troubleshooting.md

```diff
@@ -243,6 +244,17 @@ check that:
 If you get this error and none of the above applies, please
 [get in touch with us](#support).
 
+## Project does not display live metrics and plots
+
+Confirm that you are correctly following the
+[procedure to send live metrics and plots](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots)
+to Iterative Studio.
+
+Also note that live metrics and plots for an experiment are displayed only if
+its parent Git commit is present in the project table. So, before you run the
+experiment, make sure that its parent commit is pushed to Git and shown in the
```

Comment:
> before you run the experiment

Here we mean `dvc exp run` right? Or `python` it having DVCLive setup I guess... but either way run locally, not from Studio?

Reply:
Hmmm. Could this problem happen with experiments run from Studio?

Reply:
Not currently, coz when an experiment is run from Studio, we always create a Git commit. But, this can change in the future, with Lightweight experiments and other changes.